x-pack/platform/plugins/shared/alerting/server/routes/alert_delete/apis/last_run/get_alert_delete_last_run_route.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { IRouter } from '@kbn/core/server';
+import type { AlertDeleteLastRunResponseV1 } from '../../../../../common/routes/alert_delete';
+import type { ILicenseState } from '../../../../lib';
+import type { AlertingRequestHandlerContext } from '../../../../types';
+import { INTERNAL_BASE_ALERTING_API_PATH } from '../../../../types';
+import { verifyAccessAndContext } from '../../../lib';
+import { API_PRIVILEGES } from '../../../../../common';
+import { transformAlertDeleteLastRunToResponse } from '../../transforms';
+
+export const alertDeleteLastRunRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.get(
+    {
+      path: `${INTERNAL_BASE_ALERTING_API_PATH}/rules/settings/_alert_delete_last_run`,
```

Comment:
nit: could we create an `INTERNAL_ALERTING_DELETE_LAST_RUN` or something? That is the logic followed for alerting APIs in `x-pack/platform/plugins/shared/alerting/common/index.ts`, where `INTERNAL_BASE_ALERTING_API_PATH` comes from.

`ctrl + f` shows 5 places where this URL is used, I think it would make sense to replace them by a constant.

Reply:
If I have time, I'll add a new PR to do so also for the other endpoints related to alert deletion